Tasks/AzureAppServiceSettingsV1/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vsts-tasks-azureappservicesettings",
+  "version": "1.0.0",
+  "description": "Azure App Service Settings",
+  "main": "azurewebappsettings.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "2.2.48",
+    "@types/node": "6.0.68",
+    "@types/q": "1.0.7",
+    "azurermdeploycommon": "file:../../_build/Tasks/Common/azurermdeploycommon-1.1.0.tgz",
```

Comment:
instead of consuming entire azurermdeploy-common, consume only azure-arm-restv2

Reply:
We are also using azurermdeploycommon/operations here.